tests/functional/ext/docparams/parameter/missing_param_doc_required_Numpy.py

```diff
@@ -0,0 +1,6 @@
+"""Tests for missing-param-doc and missing-type-doc for Numpy style docstrings
```

Comment:
I'm curious, do you create this commit at the very end by rebasing and taking the value you had after moving everything ?

Reply:
Taken from `missing_param_doc_required_Google` as they tend to have the same tests and thus similar ignores. I don't think there are any unnecessary ignores in here (and it doesn't really hurt if there are as long as they are not the messages we want to test)

Reply:
Ho alright, thank you for explaining :)